python/ray/tempfile_services.py

```diff
@@ -0,0 +1,242 @@
+import binascii
+import datetime
+import logging
+import os
+import shutil
+import tempfile
+
+import ray.utils
+
+logger = logging.getLogger(__name__)
+
+RAY_TEMPDIR = 'RAY_TEMPDIR'
+RAY_OBJECT_STORE_SOCKET_NAME = 'RAY_OBJECT_STORE_SOCKET_NAME'
```

Comment:
Could we have command line arguments configuring this, not environment variables?

Reply:
Let's not use environment variables. If these are needed, please put them in `ray_constants.py`